dump/README.md

```diff
@@ -0,0 +1,45 @@
+# Dump books into a bucket
+
+This is a script to dump the contents of the following books of interest into a specified S3 bucket.
+
+## Books of interest
+
+1. Elementary Algebra (col12116) - 0889907c-f0ef-496a-bcb8-2a5bb121717f@8.42
+1. Intermediate Algebra (col12119) - 02776133-d49d-49cb-bfaa-67c7f61b25a1@8.13
+1. Prealgebra (col11756) - caa57dab-41c7-455e-bd6f-f443cda5519c@19.5
+
+## Installation
+
+```sh
+pip install -r requirements.txt
+```
+
```

Comment:
May be we can add a link to boto3 quick start for the aws set up stuff https://github.com/boto/boto3#quick-start